docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,261 @@
+   #. At **step 3**, choose the 18-word length and enter your Insolar Wallet's backup phrase.
+   
+#. (Ledger Nano S only) `Install the latest firmware on your device <https://support.ledger.com/hc/en-us/articles/360002731113-Update-Ledger-Nano-S-firmware>`_.
+
+Next, install the Insolar application on your device as described below.
+
+.. _install-ins-app:
+
+Install the Insolar application on Ledger Nano
+----------------------------------------------
+
+To install the Insolar application on Ledger Nano:
+
+#. Open Ledger Live, go to :guilabel:`Settings` > :guilabel:`Experimental features`, and enable :guilabel:`Developer mode`.
+
+   .. image:: imgs/mig-test/ledger-dev-mode.png
+      :width: 700px
+
+#. Open the :guilabel:`Manager` tab and connect and unlock your Ledger Nano.
+
+   .. image:: imgs/mig-test/ledger-live-connect.png
+      :width: 600px
+
+#. If prompted, press both :guilabel:`left` and :guilabel:`right` buttons simultaneously on the device to allow the manager connection.
+
+   .. image:: imgs/mig-test/allow-ledger-live.png
+      :width: 300px
+
+#. Find :guilabel:`Insolar` in the application catalog and click :guilabel:`Install` next to it.
+
+   This displays the installation window with a progress bar. Wait for the installation to complete.
+
+#. In the dashboard of the Ledger Nano device, press :guilabel:`left` or :guilabel:`right` buttons to find the Insolar application.
+
+#. Once found, press both :guilabel:`left` and :guilabel:`right` buttons simultaneously to launch the application.
+
+Once the Insolar application is launched, proceed to creating an Insolar Wallet if you don't have one. Otherwise, :ref:`log in your wallet <log-in-nano>`.
+
+Create a connected Insolar Wallet
+-----------------------------------
+
+To create an Insolar Wallet using the Insolar application on Ledger Nano, go through the following steps:
+
+#. In Google Chrome, open the `Insolar Wallet <https://wallet.insolar.io>`_ and click :guilabel:`CREATE A NEW WALLET`.
+
+   .. image:: imgs/mig-test/create-ins-wlt.png
+      :width: 400px
+
+#. On the **Create a new Wallet** screen, click :guilabel:`USE LEDGER NANO S`.
+
+   .. image:: imgs/mig-test/use-ledger-n.png
+      :width: 400px
+
+#. Make sure your Ledger Nano is connected, unlocked, and the Insolar application is launched on it.
+
+   .. _enter_key_number:
+
+#. If required, enter the key number. Ledger Nano can store multiple private keys—each to an individual Insolar Wallet. Every key stored in the device has a number. By default, the number of the first key is ``0``.
+
+   .. important:: Remember the number of this private key. You are required to specify it upon every login to use a particular Insolar MainNet Wallet.
+
+   .. image:: imgs/mig-test/key-number.png
+      :width: 500px
+
+#. Check the boxes to allow anonymous data collection and agree to the terms of use and click :guilabel:`CONNECT TO LEDGER NANO S`.
```

Comment:
Check the boxes to allow anonymous data collection and agree to the terms of **use. Then** click :guilabel:`CONNECT TO LEDGER NANO S`.